packages/docs/docs/getting-started/layouts.mdx

```diff
@@ -107,6 +107,65 @@ useful properties are listed below:
 <ApiSnippet url={'/api/2d/components/Layout#justifyContent'} />
 <hr />
 
+### Disabling `layout` on a child node
+
+Sometimes it can be useful to remove the node from the node tree. This way the
+node can be styled independently of its ancestor layout, as well as moved freely
+on the canvas.
+
+Removal of node from the node tree is done by setting a value of `layout`
+property to `false`. The node can be brought back to its original position
+inside the node tree by setting the value of `layout` property to `true`.
+
+After disabling `layout` on a node, the following things change:
+
+1. the node is moved to the place of origin, e.g. center of the scene,
```

Comment:
That's not true. The node's position will be whatever the `position` property is set to. Enabling the layout does not change this property - it overrides it but the values are still kept in there.